src/OSmOSE/utils/formatting_utils.py

```diff
@@ -13,19 +13,23 @@
 
 
 def aplose2raven(
-    df: pd.DataFrame, audio_datetimes: list[pd.Timestamp], audio_durations: list[float]
+    df: pd.DataFrame,
+    audio_datetimes: list[pd.Timestamp],
+    audio_durations: list[float],
 ) -> pd.DataFrame:
-    """
-    Exports an APLOSE formatted result file to Raven formatted DataFrame
-    provided the list of audio files / durations considered for the Raven campaign.
+    r"""Format an APLOSE result DataFrame to a Raven result DataFrame.
+
+    The list of audio files and durations considered for the Raven campaign should be
+    provided to account for the deviations between the advertised and actual
+    file durations.
 
     Parameters
     ----------
     df: Dataframe,
         APLOSE formatted result DataFrame.
 
     audio_datetimes: list[pd.Timestamp]
-        list of non-naïve timestamps from considered audio files.
+        list of tz-aware timestamps from considered audio files.
 
     audio_durations: list[float]
         list of all considered audio file durations.
```

Comment:
I guess we use `float` instead of `Timedelta` because it's the Aplose output format? 
If so, I would have specified that the `audio_durations` represent **seconds** in the docstring.